pytorch_to_returnn/torch/nn/modules/linear.py

```diff
@@ -60,6 +60,27 @@ def import_params_torch_to_returnn(self, *, layer: LinearLayer, torch_module: Li
       layer.params["b"].load(torch_module.bias.detach().numpy(), session=session)
 
 
+class FunctionalLinear(Module):
+  is_original_torch_module = False
+
+  def __init__(self):
+    super(FunctionalLinear, self).__init__()
+
```

Comment:
```suggestion
```

Reply:
Is there a difference?

Reply:
Functionality-wise, no, but it's cleaner.